packages/gatsby-source-wordpress/docs/plugin-options.md

```diff
@@ -63,6 +63,9 @@
   - [presets[].presetName](#presetspresetname)
   - [presets[].useIf](#presetsuseif)
   - [presets[].options](#presetsoptions)
+- [searchAndReplace](#searchandreplace)
```

Comment:
This file is auto-generated. `src/steps/declare-plugin-options-schema.ts` will need to be updated instead and then `npm run generate-plugin-options-docs` will make these changes automatically.

Reply:
Moved the options to the options schema declaration.

I had to replace the literal expression with a string as prettier somehow couldn't handle the literal expression, and Joi doesn't seem to support such type.